projects/1.20/assets/argentinasdelight/argentinas_delight/lang/zh_cn.json

```
@@ -0,0 +1,566 @@
+  "block.cambiandoversion.chori_3click": "烤辣肉肠 3",
+  "block.cambiandoversion.choricrudobloque": "生辣肉肠",
+  "block.cambiandoversion.chorihechobloque": "熟辣肉肠",
+  "block.cambiandoversion.hogueraparrilla": "篝火烤架",
+  "block.cambiandoversion.hojasdemate": "马黛树叶",
+  "block.cambiandoversion.jarradoradabloque": "一扎闪烁的西瓜汁",
+  "block.cambiandoversion.jarraglowberry": "一扎荧光果汁",
+  "block.cambiandoversion.jarragua": "一扎水",
+  "block.cambiandoversion.jarrahielobloque": "一扎冰水",
+  "block.cambiandoversion.jarralocabloque": "一扎狂野特调",
+  "block.cambiandoversion.jarramanzana": "一扎苹果汁",
+  "block.cambiandoversion.jarrarojabloque": "一扎甜浆果汁",
+  "block.cambiandoversion.jarravacia": "玻璃扎",
+  "block.cambiandoversion.lechebloque": "奶壶",
+  "block.cambiandoversion.lechecalientebloque": "热奶壶",
+  "block.cambiandoversion.mateamargo": "苦马黛茶",
+  "block.cambiandoversion.matedulce": "甜马黛茶",
+  "block.cambiandoversion.matelechebloque": "马黛奶茶",
+  "block.cambiandoversion.matesecodulce": "放有糖与马黛茶叶的茶杯",
+  "block.cambiandoversion.matevacio": "空马黛茶杯",
+  "block.cambiandoversion.mateyerbaseca": "放有马黛茶叶的茶杯",
+  "block.cambiandoversion.pava": "铁壶",
+  "block.cambiandoversion.pavacaliente": "沸水壶",
+  "block.cambiandoversion.pavadulce": "马黛茶壶",
+  "block.cambiandoversion.pavallena": "水壶",
+  "block.cambiandoversion.pimpolloyerba": "马黛树种苗",
+  "block.cambiandoversion.polloasadobloque": "熟鸡肉",
+  "block.cambiandoversion.pollocrudobloque": "生鸡肉",
+  "block.cambiandoversion.ramasdematedeyerba": "马黛树杈",
+  "block.cambiandoversion.sandiajarra": "一扎西瓜汁",
+  "block.cambiandoversion.tablachancho_1click": "烤猪肉 1",
+  "block.cambiandoversion.tabladeasadobloque": "阿根廷烤牛肉",
+  "block.cambiandoversion.tabladechancho": "阿根廷烤猪肉",
+  "block.cambiandoversion.tabladepollo": "阿根廷烤鸡",
+  "block.cambiandoversion.tablapollo_1click": "阿根廷烤鸡 1",
+  "block.cambiandoversion.tablapollo_2click": "阿根廷烤鸡 2",
+  "block.cambiandoversion.tablapollo_3click": "阿根廷烤鸡 3",
+  "block.cambiandoversion.tablasado_1click": "阿根廷烤牛肉 1",
+  "block.cambiandoversion.tablasado_2click": "阿根廷烤牛肉 2",
+  "block.cambiandoversion.tablasado_3click": "阿根廷烤牛肉 3",
+  "block.cambiandoversion.tereberryrojabloque": "甜浆果冰马黛茶",
+  "block.cambiandoversion.terebloque": "冰马黛茶",
+  "block.cambiandoversion.teredemanzana": "苹果冰马黛茶",
+  "block.cambiandoversion.teredesandiadorada": "闪烁的西瓜冰马黛茶",
+  "block.cambiandoversion.tereglowberry": "荧光冰马黛茶",
+  "block.cambiandoversion.teresandia": "西瓜冰马黛茶",
+  "block.cambiandoversion.yuyosbloque": "马黛药草茶",
+  "block.cambiandoversion.yuyoseco": "放有药草与马黛茶叶的茶杯",
+  "effect.acido": "酸性",
+  "effect.argentinas_delight.acido": "酸性",
+  "effect.argentinas_delight.arisco": "生人勿近",
+  "effect.argentinas_delight.baya_baya": "果然危险",
+  "effect.argentinas_delight.bonanza": "繁茂",
+  "effect.argentinas_delight.buen_provecho": "祝你好胃口",
+  "effect.argentinas_delight.cargandomuertoefecto": "尸体拖拽",
+  "effect.argentinas_delight.desarrollo": "育成",
+  "effect.argentinas_delight.desconocerse": "野狂",
```

[百科页面](https://www.mcmod.cn/item/764339.html)
效果为在玩家周围播放诡异的音效，效果结束后在玩家处生成一只永远也不会长大的小村民。
```suggestion
  "effect.argentinas_delight.desconocerse": "纵情恣欲",
```

---

hmmm
beta3.0 不会出现对应效果，疑似已移除